docs/tools/flow-cli/flix.md

```diff
@@ -1,35 +1,74 @@
 ---
-title: Execute Flow Interaction Templates (FLIX) with the Flow CLI
-sidebar_title: Execute FLIX
-description: How to execute Flow Interaction Templates (FLIX) on Flow from the command line
+title: Flow Interaction Templates (FLIX)
+sidebar_title: Execute, Package FLIX
+description: Flow Interaction Templates (FLIX) on Flow from the command line
```

Comment:
needs a verb for the description like "Interact with Flow Interaction Templates (FLIX) via the CLI"

Reply:
I'm referring to the description no the sidebar title here

Reply:
gotcha, I'll add "sidebar_label" so that "title" isn't used in the sidebar and there can be a different title of the page. 
![image](https://github.com/onflow/docs/assets/3970376/a42887f3-4a34-4158-8aa8-5f67e5f6884a)

Reply:
Meant this `description: Interact with Flow Interaction Templates (FLIX) via the CLI.`